### ISSUE TITLE
compilation fails on Ubuntu 18.04

### ISSUE THREAD

**Issue by gijzelaerr** [lofar-astron/PyBDSF#57]
```
C compiler: x86_64-linux-gnu-g++ -pthread -DNDEBUG -g -fwrapv -O2 -Wall -fno-strict-aliasing -march=native -O2 -pipe -fPIC

creating build/temp.linux-amd64-2.7/src/c++
creating build/temp.linux-amd64-2.7/src/c++/num_util
compile options: '-Isrc/c++ -I/usr/lib/python2.7/dist-packages/numpy/core/include -I/usr/include/python2.7 -c'
x86_64-linux-gnu-g++: src/c++/Fitter_lmder.cc
In file included from src/c++/Fitters.h:5:0,
                 from src/c++/Fitter_lmder.cc:11:
src/c++/MGFunction.h:49:14: error: ‘numeric’ has not been declared
   MGFunction(numeric::array data, numeric::array mask, double weight);
              ^~~~~~~
src/c++/MGFunction.h:49:29: error: expected ‘)’ before ‘data’
   MGFunction(numeric::array data, numeric::array mask, double weight);
                             ^~~~
src/c++/MGFunction.h:123:3: error: ‘numeric’ does not name a type; did you mean ‘cuserid’?
   numeric::array m_data;
   ^~~~~~~
   cuserid
src/c++/MGFunction.h:125:3: error: ‘numeric’ does not name a type; did you mean ‘cuserid’?
   numeric::array m_mask;
   ^~~~~~~
   cuserid
In file included from src/c++/Fitters.h:5:0,
                 from src/c++/Fitter_lmder.cc:11:
src/c++/MGFunction.h:49:14: error: ‘numeric’ has not been declared
   MGFunction(numeric::array data, numeric::array mask, double weight);
              ^~~~~~~
src/c++/MGFunction.h:49:29: error: expected ‘)’ before ‘data’
   MGFunction(numeric::array data, numeric::array mask, double weight);
                             ^~~~
src/c++/MGFunction.h:123:3: error: ‘numeric’ does not name a type; did you mean ‘cuserid’?
   numeric::array m_data;
   ^~~~~~~
   cuserid
src/c++/MGFunction.h:125:3: error: ‘numeric’ does not name a type; did you mean ‘cuserid’?
   numeric::array m_mask;
   ^~~~~~~
   cuserid
error: Command "x86_64-linux-gnu-g++ -pthread -DNDEBUG -g -fwrapv -O2 -Wall -fno-strict-aliasing -march=native -O2 -pipe -fPIC -Isrc/c++ -I/usr/lib/python2.7/dist-packages/numpy/core/include -I/usr/include/python2.7 -c src/c++/Fitter_lmder.cc -o build/temp.linux-amd64-2.7/src/c++/Fitter_lmder.o -MMD -MF build/temp.linux-amd64-2.7/src/c++/Fitter_lmder.o.d" failed with exit status 1
E: pybuild pybuild:336: build: plugin distutils failed with: exit code=1: /usr/bin/python setup.py build 
dh_auto_build: pybuild --build -i python{version} -p 2.7 returned exit code 13
debian/rules:11: recipe for target 'build' failed
make: *** [build] Error 25
dpkg-buildpackage: error: debian/rules build subprocess returned exit status 2
```

**Comment by tammojan** [comment]
Please use the latest release, this should have been fixed with ad6cdd64e2907d03d4af2136904f4bf30a5fc38b

**Comment by gijzelaerr** [comment]
done, thanks!
